pkg/cloud/services/network/natgateways.go

```diff
@@ -69,6 +69,7 @@ func (s *Service) reconcileNatGateways() error {
 		return err
 	}
 
+	var natGatewaysIPs []string
```

Comment:
For consistency with the next line
```suggestion
	natGatewaysIPs := []string{}
```

Reply:
Alternatively, the line could change to the var declaration. :)

Reply:
Especially because it looks like it could be empty, so no need to pre-initialize it.

Reply:
Why is one way preferred over the other? Isn't it a empty slice in both cases?